File: fc/examples/test2.lua
function INIT()
	screen(320,200,4)
	print('ok')
end

function DRAW()
	cursor(8,8)
	print("hello")
	print('world',nil,16)
	print('to jest bardzo ^5dlugi^2 tekst ktory\npowinien\n^^zlamac\nlinie',100,100)
end


function POST()
	love.graphics.setColor(0,0,0,1)
	if false then for y = 4,scrh*scrs,4 do
		love.graphics.line(0,y,scrw*scrs,y)
	end end 
	if false then for x = 4,scrw*scrs,4 do
		love.graphics.line(x,0,x,scrh*scrs)
	end end
end

File: docs/itsy0/test2.js
function _init() {
}

function _main() {	
	status(`t1:${fc.t1}  t2:${fc.t2}  t3:${fc.t3}  t4:${fc.t4}`)
}

function _draw() {
	color(1)
	for (var i=0;i<500;i++) {
		for (var j=0;j<500;j++) {
			line(i,j,i,j,4)
		}
	}
}
